Documentation/Concepts/Index.rst

```diff
@@ -54,26 +54,25 @@ Backend Modules
    .. container:: col-md-8
 
       The backend contains a range of modules that are grouped by task. User
-      access rights determine what modules are visible to users when they are
-      logged into the backend.
+      access rights determine what modules are visible to users when they log into the backend.
 
       -  The Web group contains a set of modules that handle the creation and
          management of both pages and content.
 
       -  Site Management handles the setup of a site. From this module it is
-         possible to specify the site name, assign domains and select what
-         languages are to be used.
+         possible to specify the site name, assign domains and select
+         languages.
 
-      -  Filelist provides a convenient way to view and manage files including
+      -  Filelist provides a convenient way to view and manage files, including
          documents, images and videos.
 
-      -  Admin Tools features a collection of administrative modules that enable
-         you to perform various maintenance and upgrade tasks. This module also
-         contains the Extension manager, allowing you to enable and disable any
-         third-party extensions that are currently installed.
+      -  Admin Tools features a collection of administrative modules so that
+         you can perform various maintenance and upgrade tasks. This module also
```

Comment:
It's a section (or main module) but not a module according to TYPO3 specific definition.